conf/environments.js

```diff
@@ -102,6 +102,11 @@ module.exports = {
             ecmaVersion: 6
         }
     },
+    es7: {
```

Comment:
> Concern: what happens when I set ES6 environment and then ecmaVersion to 7? Which one takes precedence? I'm guessing and hoping that ecmaVersion setting takes precedence but don't recall a doc specifically mentioning this.

ES7 is used for the parser and the es6 globals are added.

I agree with @mysticatea that we probably need to rethink the relationship between version and globals in the future.